docs/src/lib/functions.md

```diff
@@ -16,7 +16,10 @@ by
 combine
 groupby
 groupindices
+groupvalues
```

Comment:
```suggestion
```

Reply:
I do not know how to force-push this. I will remove this line in 0.20 PR as this function does not exist now